pkg/policies/branch/branch.go

```diff
@@ -88,23 +97,35 @@ type RepoConfig struct {
 
 	// BlockForce overrides the same setting in org-level, only if present.
 	BlockForce *bool `yaml:"blockForce"`
+
+	// RequireUpToDateBranch : set to true to require that branches must be up
+	// to date before merging. Default false.
+	RequireUpToDateBranch *bool `yaml:"requireUpToDateBranch"`
```

Comment:
Update the comment to indicate how the repo-config is merged with the org-config. Ex:
`RequireUpToDateBranch overrides the same setting in org-level, only if present.`

Reply:
Fixed in 754144bd3f9e24f3554793450990d44d4116b98f